### PR TITLE
Fixes, enhancements, second menu 

### PR DESCRIPTION
**Pull request by coderbloke**
Hello
Just some small fixes
(Edit some days later: Not only some small fixes anymore. Read forward and see teaser.)

### CODE REVIEW

**Comment by coderbloke** [comment]
Hmm. After the first comment, I made another update, which is more fancy. I don't know how to make separate pull request. Never mind.
It got an icon also. Made by me under CC0 (i.e. I don't care) license. I may change it a bit later, as I plan to make same compact view for an another plug-in enabler Godot plug-in. In that case I want to differentiate between the 2 in icon also.

**Comment by coderbloke** [comment]
Instead of updating the other plug-in also, I added the idea from that into this.
That was a fork of an very old unmaintained plug-in, and basically this one had almost all function, which was needed to add this second type of menu.
Teaser: https://drive.google.com/file/d/1bbZ7E5IpQl_cJWnG2d5jEfRhCc75THoH/view?usp=sharing

**Comment by mrpedrobraga** [comment]
Greetings, @coderbloke !
It took me a time to notice this PR as this repository isn't very full of people; I'll take a look at it later today and possibly merge it;

**Comment by hollyfromthenexus** [comment]
I'll also take a peek at this, the plugin subfolder fix is directly relevant to me :)

**Comment by hollyfromthenexus** [comment]
Can confirm, functionally this PR works for my use-case! The new UI options are also pretty clean :D

**Comment by mrpedrobraga** [comment]
image

This is stellar, @coderbloke made it look like it should be part of the engine;
There are a few regressions, and some leftover things which I'll clean up and soon after merge the PR;

Most notable regression, is, refreshing a plugin doesn't persist which Main Screen you had open... but I'll address that on another PR;

I'd normally @coderbloke if they'd like to do something before I merge this PR to main, but their last contribution was in May, 29, which was a long time ago.

File: addons/plugin_refresher/refresh_button.gd [hunk 1]
```diff
@@ -1,98 +1,433 @@
...
+			switch_options.icon = null
 
+# Currently not implemented anywhere,
+# It is useful, for writing plugins
+# which have main screens, to keep the same
+# tab selected across reloads.
+# The main screen tab tends to change
+# because the plugin's tab ceases to exist
+# when it is deactivated.
 func get_main_screen()->String:
```
Comment (mrpedrobraga) [inline addons/plugin_refresher/refresh_button.gd:428]: This became unused; I'll reimplement this in a future PR.

**Review by mrpedrobraga — APPROVED** [review]

**Pull request merged**